Brokerages/InteractiveBrokers/InteractiveBrokersBrokerage.cs

```diff
@@ -959,6 +959,169 @@ private IEnumerable<ContractDetails> FindContracts(Contract contract)
             return contractDetails;
         }
 
+        /// <summary>
+        /// Gets the current conversion rate into Account currency
+        /// </summary>
+        /// <remarks>Synchronous, blocking</remarks>
+        private decimal GetCurrencyConversion(string currency)
```

Comment:
Did we simply copy the `GetUsdConversion` method instead of upgrading it? The purpose of `GetUsdConversion` is actually to get the conversion in terms of the account currency, so I don't think we need to versions of this same method.